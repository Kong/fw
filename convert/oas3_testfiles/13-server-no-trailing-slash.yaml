# servers[0].url doesn't have a trailing slash
# service.url should be "/"" not ""
openapi: 3.0.3
info:
  title: Server with no trailing slash
  version: v1
servers:
  - url: "https://example.com"

paths:
  /demo:
    get:
      tags: [demo]
      operationId: list
      description: This is a demo OAS
      responses:
        "200":
          description: OK
          content:
            "application/json":
              schema:
                properties:
                  status:
                    type: string
tags:
  - name: demo
