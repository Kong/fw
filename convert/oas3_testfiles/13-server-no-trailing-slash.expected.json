{
  "_format_version": "3.0",
  "services": [
    {
      "host": "example.com",
      "id": "5434f850-a000-5ad5-8772-1a3cd10811e6",
      "name": "server-with-no-trailing-slash",
      "path": "/",
      "plugins": [],
      "port": 443,
      "protocol": "https",
      "routes": [
        {
          "id": "7d6c6daa-9b04-5e64-8903-0ecd06e02565",
          "methods": [
            "GET"
          ],
          "name": "server-with-no-trailing-slash_list",
          "paths": [
            "~/demo$"
          ],
          "plugins": [],
          "strip_path": false,
          "tags": [
            "OAS3_import",
            "OAS3file_13-server-no-trailing-slash.yaml"
          ]
        }
      ],
      "tags": [
        "OAS3_import",
        "OAS3file_13-server-no-trailing-slash.yaml"
      ]
    }
  ],
  "upstreams": []
}